src/uwtools/set_config.py

```diff
@@ -87,6 +95,8 @@ def create_config_obj(argv):
 
     else:
         print("Set config failure: bad file type")
+        log.info(f'warning file {user_args.input_base_file} ',
```

Comment:
This f-string was giving me a hard time on my copy, but if it passes pytest its ok

Reply:
This logged output is the culprit for the failing test because it's testing only for output that says "Set config failure: bad file type"

 I believe if you change the log to:

```
log.info("Set config failure: bad file type")
```
The test will pass.

Alternatively, if you want to keep the more verbose log message (modified to be one string message), the test will need to be updated to match the new output string.

I would suggest the latter since you've already gone to the trouble of introducing the logger and defining a verbose message introduced a logger.